pkg/network/link/discovery.go

```diff
@@ -45,6 +45,28 @@ func DiscoverByNetwork(handler driver.NetworkHandler, networks []v1.Network, sub
 	return linkByNames(handler, ifaceNames)
 }
 
+func DiscoverByDefaultGateway(family int) ([]netlink.Link, error) {
```

Comment:
Usually I don't bother with function documentation, but since it touches with a relatively low level wrapper, please consider adding one, explaining what this function does.